ls1intum/Artemis
# Bugfix/client/tooltip not released placement

### Checklist

### Motivation and Context


Fix one more client styling issue

### Description

Fixes the placement of the tooltip shown when hovering over "not released" tags.   Also fixed the ngtooltip placement issue in the exam scores page.
<img src=https://user-images.githubusercontent.com/33342534/124032608-10fc8200-d9f9-11eb-821d-77cf744a5a00.png width=300>

<img src=https://user-images.githubusercontent.com/33342534/124032367-c8dd5f80-d9f8-11eb-84a2-4ed461ba7f55.png width=300>


### Steps for Testing


1. Log in to Artemis
2. Navigate into one course with unreleased exercises
3. Hover over the "not released" tag and check the placement of the tooltip

### Screenshots
